web/src/features/charts/OriginChart.tsx

```diff
@@ -24,7 +24,7 @@ interface OriginChartProps {
   timeRange: TimeRange;
 }
 
-// TODO(cady): fix types to use ElectricityModeType
+// TODO: fix types to use ElectricityModeType
```

Comment:
😅

Reply:
I'm just embarrassed that it slipped by me and made it into the codebase 🤦

Reply:
Don't be, all my TODOs go into the codebase...

Hopefully I'll fix them all at some point 😅